src/participant/docsChatbotAIService.ts

```diff
@@ -0,0 +1,130 @@
+import type { Reference, VerifiedAnswer } from 'mongodb-rag-core';
+
+const MONGODB_DOCS_CHATBOT_API_VERSION = 'v1';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { version } = require('../../package.json');
+
+export type Role = 'user' | 'assistant';
+
+export type ConversationData = {
+  _id: string;
+  createdAt: string;
+  messages: MessageData[];
+  conversationId: string;
+};
+
+export type MessageData = {
+  id: string;
+  role: Role;
+  content: string;
+  createdAt: string;
+  rating?: boolean;
+  references?: Reference[];
+  suggestedPrompts?: string[];
+  metadata?: AssistantMessageMetadata;
+};
+
+export type AssistantMessageMetadata = {
```

Comment:
Do we use these types anywhere external to this file? If not can we avoid exporting them? That way we know if they're no longer used.
```suggestion
type Role = 'user' | 'assistant';

type ConversationData = {
  _id: string;
  createdAt: string;
  messages: MessageData[];
  conversationId: string;
};

type MessageData = {
  id: string;
  role: Role;
  content: string;
  createdAt: string;
  rating?: boolean;
  references?: Reference[];
  suggestedPrompts?: string[];
  metadata?: AssistantMessageMetadata;
};

type AssistantMessageMetadata = {
```